bpfctl/src/main.rs

```diff
@@ -105,7 +105,8 @@ async fn main() -> anyhow::Result<()> {
         .domain_name("localhost")
         .ca_certificate(ca_cert)
         .identity(identity);
-    let channel = Channel::from_static("http://[::1]:50051")
+    let address = String::from("http://") + config.grpc.endpoint.address.as_str();
```

Comment:
I know that the protocol was already here previously, but do do we have an issue to follow up and switch this to HTTPS?

Might be good to link an issue as a TODO here?

Reply:
I can't see any issues for switching over to https, maybe more of a question for maintainers since I'm not that well versed in the code base and roadmap.

Reply:
@astoycos you think we should get drop a reminder here while we're making changes so we don't forget to encrypt this before (maybe?) `v0.2.0`?

Reply:
I don't think we have an issue to move to https, just #317, so I suggest writing one.

Reply:
Created a placeholder issue (#396) and added a TODO comment.